pkgs/development/python-modules/starlette/default.nix

```diff
@@ -46,12 +46,15 @@ buildPythonPackage rec {
   checkInputs = [
     aiosqlite
     databases
+    pytest-asyncio
+    pytest-cov
     pytestCheckHook
     typing-extensions
   ];
 
   pytestFlagsArray = [ "--ignore=tests/test_graphql.py" ];
```

Comment:
```suggestion
  disabledTestFiles = [ "tests/test_graphql.py" ];
```

Reply:
Do note, that until #112095 is merged this will not work for multiple files or file paths that have spaces.

Reply:
Switched to `disabledTestFiles`.